delft3dgtprotectedservices/views.py

```diff
@@ -64,6 +64,7 @@ def thredds_static(request, loc):
 
     # redirect to nginx thredds
     response = HttpResponse()
-    response["X-Accel-Redirect"] = "/protected_thredds/{0}".format(loc)
+    response["X-Accel-Redirect"] = "/protected_thredds/{0}?{1}".format(
```

Comment:
Ah! As you can see, previously this view threw away the GET params in the redirect. This is not directly related to the FE performance bug, but I wanted this included in this hotfix as Thredds uses a lot of GET params to work properly.